src/api/messages/getMessages.js

```diff
@@ -56,11 +56,11 @@ export const migrateMessages = (messages: ServerMessage[]): Message[] =>
     };
   });
 
-const migrateResponse = response => {
+const migrateResponse = (response, auth: Auth) => {
```

Comment:
Let's pass an `Identity` instead. From the `Auth` jsdoc:
```
 * Prefer using `Identity` where possible, which identifies the user but
 * leaves out the secret API key; use `identityOfAuth` to extract one from
 * an `Auth`.
```

(Or passing `auth.realm` would be good too.)